docs/advanced.rst

```diff
@@ -348,3 +348,21 @@ If you want to save a model without a historical record, you can use the followi
 
     poll = Poll(question='something')
     poll.save_without_historical_record()
+
+History Diffing
+-------------------
+
+When you have two instances of the same HistoricalRecord (such as the HistoricalPoll example above),
+you can perform diffs to see what changed. This will result in a ModelDelta containing properties including
+a list with each field changed between the two historical records, a list with the names of all
+fields that incurred changes from one record to the other, and the old and new records.
+This may be useful when you want to construct timelines and need to get only the model modifications.
+
+.. code-block:: python
+
+    p = Poll.objects.create(question="what's up?")
+    p.question = "what's up, man?"
+    p.save()
+
+    new_record, old_record = p.history.all()
+    delta = new_record.diff_against(old_record)
```

Comment:
Might want to add some more code examples that show what that `delta` object contains – I wouldn't know what it has without looking through the code or inspecting the instance